### DISCUSSION TITLE
Support for Installation from Windows package managers

### DISCUSSION THREAD

**Discussion by debajyoti1990** [VladimirMarkelov/ttdl#78] [General]
scoop/ winget/ chocolatey package managers provide easy installation and auto-addition to path..

**Comment by VladimirMarkelov** [comment]
That's a good idea. I was thinking about it for a while but I have done nothing yet. I use `scoop` to install some utilities, so I am familiar a bit with it. It looks like I can add TTDL only to `extra` bucket as it does not meet `main` bucket requirements. As of `chocolately` and `winget` I have no clue yet what the requirements and procedure of registering are .
